packages/reports/addon/components/navi-column-config.js

```diff
@@ -115,7 +114,7 @@ class NaviColumnConfig extends Component {
         });
     }
 
-    return null;
+    return columns.length === 1 && column.type === 'timeDimension' ? columns[0] : null;
```

Comment:
Oops that should've been 
```suggestion
    return columns.length === 1 && columns[0].type === 'timeDimension' ? columns[0] : null;
```